content/en/checkout/guide.md

```diff
@@ -79,11 +79,18 @@ This return object is referred to as a **Checkout manifest**, and will be needed
 
 ### Script tag
 
-Include the easypay Checkout SDK in your page:
-
-```html
-<script src="https://cdn.easypay.pt/checkout/2.1.0/"></script>
-```
+Include the easypay Checkout SDK in your page
+
+You have 2 options:
+
+- For test environment:
+  ```html
+  <script src="https://cdn.quality-utility.aws.easypay.pt/checkout/2.1.0"></script>
+  ```
+- For production environment:
+  ```html
+  <script src="https://cdn.easypay.pt/checkout/2.1.0"></script>
+  ```
```

Comment:
I think the SDK itself should always be included from the production build. It already has a flag for testing.
Also, the quality-utility environment is theoretically secret to us.

Reply:
I can remove it then.